reports/__init__.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "0.1.8"
+__version__ = "0.0.10"
```

Comment:
@jthiles any reason this was not synced with the package version in https://github.com/AdvancedThreatAnalytics/django-libreport/pull/48/files#diff-108e7420e50ba79329c0d6f4e15e055eeec22ea03ee12bcbe0d08d9ccad5a4beL1 ?
I see no point why would this be different

Reply:
I don't know why it should be different and I'm not sure why/how to make it sync automatically.